Pocket/pocket-ios
# fix(load): removing extra saves & archive call

## Summary
I noticed that archive & saves was fetching twice on app launch.

## Test Steps
* Watch the logs on app launch and ensure that FetchSaves and FetchArchive is only requested once (after the initial load and its not paginating)

## PR Checklist:
- [] Added Unit / UI tests

## Screenshots
